views/index.html.twig

```diff
@@ -34,8 +34,8 @@
             {% if name %}{{ name }}{% else %}Composer repository{% endif %}
         </h1>
 
-        <small id="lastupdate">
-            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::ISO8601')) }}">{{ "now"|date(constant('\DateTime::RSS')) }}</time>
+        <small>
+            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::RFC3339')) }}">{{ "now"|date(constant('\DateTime::RFC3339')) }}</time>
```

Comment:
Well, the existing DATE_RSS is fine too, but the full day name of DATE_COOKIE is indeed more readable. On the other hand, DATE_COOKIE displays ``12-Aug-2013`` while DATE_RSS uses ``12 Aug 2013``, so it would win there